src/OnboardingSPA/utils/analytics/hiive/OnboardingEvent.js

```diff
@@ -0,0 +1,20 @@
+import { HiiveEvent } from '@newfold-labs/js-utility-ui-analytics';
+import { getLabelKeyFromAction } from '.';
+import { CATEGORY } from './constants';
+
+export class OnboardingEvent extends HiiveEvent {
+	constructor( action, value, additionalData ) {
```

Comment:
Does this run for ACTION_PAGEVIEW and all with **label_key** as undefined and no additionalData?

Reply:
It does, power of JavaScript 💪